network/channel_reassigner/network.go

```diff
@@ -0,0 +1,25 @@
+package channel_reassigner
+
+import (
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/network"
+)
+
+type ChannelReassignerNetwork struct {
```

Comment:
actually, this job is better done by a custom `subscription manager` than a network which remaps channel on `Register` reuest.